modules/ldap_contacts/modules.php

```diff
@@ -316,10 +316,11 @@ public function process() {
 class Hm_Handler_load_ldap_settings extends Hm_Handler_Module {
     public function process() {
         $connections = array();
-        foreach (ldap_config($this->config) as $name => $vals) {
-            if (array_key_exists('auth', $vals) && $vals['auth']) {
-                if ((!array_key_exists('user', $vals) || !$vals['user']) &&
-                    (!array_key_exists('pass', $vals) || !$vals['pass'])) {
+        $ldap_config = ldap_config($this->config);
+        foreach ($ldap_config as $name => $vals) {
+            if (array_key_exists('auth', $ldap_config) && $ldap_config['auth']) {
```

Comment:
I don't think this is the right way to fix the issue. Note that previously, Cypht supported multiple sets of ldap_config arrays named differently (e.g. Personal, etc.). We need to keep the same behavior with .env migration. E.g. by default, we put the initial ldap config in Personal namespace (array key) and then allow user to add more configs if needed. Then, the loops here in ldap_contacts module shouldn't be changed.

Reply:
Copy that @kroky, By updating ldap.php like this:
```
 'ldap' => [
        'Personal' => [....]
  ]
```
It works, I changed also Carddav.php to use the same structure.

Reply:
Great, thanks!